aidants_connect_web/views/espace_responsable.py

```diff
@@ -214,6 +217,50 @@ def remove_card_from_aidant(request, aidant_id):
     )
 
 
+@require_http_methods(["GET", "POST"])
+@login_required
+@user_is_responsable_structure
+@activity_required
+def remove_aidant_from_organisation(
+    request: HttpRequest, aidant_id: int, organisation_id: int
+) -> HttpResponse:
+    responsable: Aidant = request.user
+    aidant: Aidant = get_object_or_404(Aidant, pk=aidant_id)
+    organisation: Organisation = get_object_or_404(Organisation, pk=organisation_id)
+
+    if not responsable.can_see_aidant(aidant):
+        django_messages.error(
+            request,
+            f"L'aidant ou aidante avec l'identifiant numéro {aidant_id} ne fait pas "
+            f"partie de {organisation.name}",
+        )
+        return redirect(
+            "espace_responsable_organisation", organisation_id=organisation.id
+        )
+
+    if request.method == "GET":
+        return render(
+            request,
+            "aidants_connect_web/espace_responsable/"
+            "confirm-remove-aidant-from-organisation.html",
+            {"aidant": aidant, "organisation": organisation},
+        )
+
+    result = aidant.remove_user_from_organisation(organisation)
```

Comment:
Ça suffirait comme ça, non ?

```suggestion
    result = aidant.remove_from_organisation(organisation)
```